src/routes/routes.tsx

```diff
@@ -80,6 +82,9 @@ const routes = createBrowserRouter([
     ],
   },
 
+  { path: '/rooms', element: <BookingRoom /> },
```

Comment:
remove this

Reply:
why, we don't need if anymore?

Reply:
because booking will be inside the accommodation page, he created the booking page for test purpose